<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pumpsettingsscreen</class>
 <widget class="QWidget" name="pumpsettingsscreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>430</width>
     <height>414</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page0">
    <widget class="QLabel" name="pumpSettingsLabel">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>411</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Options</string>
     </property>
    </widget>
    <widget class="QPushButton" name="upButton">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>120</y>
       <width>61</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>▲</string>
     </property>
    </widget>
    <widget class="QPushButton" name="downButton">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>190</y>
       <width>61</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>▼</string>
     </property>
    </widget>
    <widget class="QPushButton" name="homeButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>←</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>281</width>
       <height>307</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="stopInsulinButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Stop Insulin</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="myPumpButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Profiles / My Pump</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="myCgmButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>My CGM</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="tempRateButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Temp Rate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loadButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="logsButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Current Status and System / CGM Logs</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deviceSettings">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Device Settings</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="changeExistingPasswordScreen">
    <widget class="QPushButton" name="confirmPinCheckBox">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>110</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 35pt &quot;DejaVu Sans&quot;;
background-color: rgb(98, 160, 234);
</string>
     </property>
     <property name="text">
      <string>✓</string>
     </property>
    </widget>
    <widget class="QLabel" name="page9MainLabel">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>80</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter Units</string>
     </property>
    </widget>
    <widget class="QPushButton" name="page9CancelButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 35pt &quot;DejaVu Sans&quot;;
background-color: rgb(246, 97, 81);
</string>
     </property>
     <property name="text">
      <string>←</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="page9TextBox">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>110</y>
       <width>104</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>160</y>
       <width>254</width>
       <height>184</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="pinButton1">
        <property name="styleSheet">
         <string notr="true">font: 75 20pt &quot;DejaVu Sans&quot;;</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pinButton2">
        <property name="styleSheet">
         <string notr="true">font: 75 20pt &quot;DejaVu Sans&quot;;</string>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pinButton3">
        <property name="styleSheet">
         <string notr="true">font: 75 20pt &quot;DejaVu Sans&quot;;</string>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pinButton4">
        <property name="styleSheet">
         <string notr="true">font: 75 20pt &quot;DejaVu Sans&quot;;</string>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pinButton5">
        <property name="styleSheet">
         <string notr="true">font: 75 20pt &quot;DejaVu Sans&quot;;</string>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pinButton6">
        <property name="styleSheet">
         <string notr="true">font: 75 20pt &quot;DejaVu Sans&quot;;</string>
        </property>
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pinButton7">
        <property name="styleSheet">
         <string notr="true">font: 75 20pt &quot;DejaVu Sans&quot;;</string>
        </property>
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pinButton8">
        <property name="styleSheet">
         <string notr="true">font: 75 20pt &quot;DejaVu Sans&quot;;</string>
        </property>
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pinButton9">
        <property name="styleSheet">
         <string notr="true">font: 75 20pt &quot;DejaVu Sans&quot;;</string>
        </property>
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="pushButton_PlusEquals">
        <property name="styleSheet">
         <string notr="true">font: 75 20pt &quot;DejaVu Sans&quot;;</string>
        </property>
        <property name="text">
         <string>.</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="pinButton0">
        <property name="styleSheet">
         <string notr="true">font: 75 20pt &quot;DejaVu Sans&quot;;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="pinBackButton">
        <property name="styleSheet">
         <string notr="true">font: 75 15pt &quot;DejaVu Sans&quot;;</string>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="confirmPinScreen"/>
   <widget class="QWidget" name="deviceSettingsPage">
    <widget class="QPushButton" name="homeButton_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>←</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>281</width>
       <height>241</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="securityPinButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Security PIN</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deviceInfoButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Device Info</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="goToSetPinButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>70</y>
       <width>111</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>✓</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="setNewPinScreen">
    <widget class="QLabel" name="newPinLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>281</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter New PIN</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="confirmNewPinScreen">
    <widget class="QLabel" name="existingLabel_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>281</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Re-enter Pin</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
